data/mods/Magiclysm/enchanted/enchanted_rings.json

```diff
@@ -0,0 +1,277 @@
+[
+  {
+    "abstract": "mring_copper",
+    "type": "TOOL_ARMOR",
+    "name": "copper magic ring",
+    "description": "A generic copper magic ring.",
+    "weight": "45 g",
+    "volume": 0,
+    "price": 5000,
+    "material": [ "copper" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "HAND_EITHER" ],
+    "coverage": 0,
+    "warmth": 0,
+    "flags": [ "WATER_FRIENDLY", "ALLOWS_NATURAL_ATTACKS", "FANCY", "ONLY_ONE", "SKINTIGHT" ]
+  },
+  {
+    "abstract": "mring_silver",
+    "type": "TOOL_ARMOR",
+    "name": "magic ring",
+    "description": "A generic silver magic ring.",
+    "weight": "5.4 g",
+    "volume": 0,
+    "price": 5000,
+    "material": [ "silver" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "HAND_EITHER" ],
+    "coverage": 0,
+    "warmth": 0,
+    "flags": [ "WATER_FRIENDLY", "ALLOWS_NATURAL_ATTACKS", "FANCY", "ONLY_ONE", "SKINTIGHT" ]
+  },
+  {
+    "abstract": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "name": "magic ring",
+    "description": "A generic gold magic ring.",
+    "weight": "9 g",
+    "volume": 0,
+    "price": 5000,
+    "material": [ "gold" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "HAND_EITHER" ],
+    "coverage": 0,
+    "warmth": 0,
+    "flags": [ "WATER_FRIENDLY", "ALLOWS_NATURAL_ATTACKS", "FANCY", "ONLY_ONE", "SKINTIGHT" ]
+  },
+  {
+    "abstract": "mring_platinum",
+    "type": "TOOL_ARMOR",
+    "name": "magic ring",
+    "description": "A generic platinum magic ring.",
+    "weight": "11.7 g",
+    "volume": 0,
+    "price": 5000,
+    "material": [ "platinum" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "HAND_EITHER" ],
+    "coverage": 0,
+    "warmth": 0,
+    "flags": [ "WATER_FRIENDLY", "ALLOWS_NATURAL_ATTACKS", "FANCY", "ONLY_ONE", "SKINTIGHT" ]
+  },
+  {
+    "copy-from": "mring_silver",
+    "type": "TOOL_ARMOR",
+    "id": "mring_blades_lesser",
+    "name": "ring of blades",
+    "name_plural": "rings of blades",
+    "description": "An ornate silver ring engraved with daggers that conjures a near perfect throwing knife into your hand on activation.",
+    "use_action": { "type": "cast_spell", "spell_id": "conj_throwing_blade1", "no_fail": true, "level": 1, "need_worn": true },
+    "initial_charges": 5,
+    "charges": 5,
+    "max_charges": 5,
+    "charges_per_use": 1,
+    "artifact_data": { "charge_type": "ARTC_TIME" }
+  },
+  {
+    "id": "conj_throwing_blade1",
+    "type": "SPELL",
```

Comment:
i'd prefer that spells stay in the spell folder. i believe there is already a json file for spells activated from objects, as well.

Reply:
I just kept it there for reference, it'll be removed and split off into a file.